tests/integration.rs

```diff
@@ -57,7 +58,17 @@ pub fn netflix() {
         supported_sig_algs: ALL_SIGALGS,
     };
 
-    assert_eq!(options.verify_trusted(&cert, time), Ok(()));
+    let path = options.verify_trusted(&cert, time).unwrap();
+    assert_eq!(path.anchor().subject, anchor.subject);
+
+    let inters = path.intermediate_certificates().collect::<Vec<_>>();
+    let mut reversed = Vec::with_capacity(inters.len());
+    let mut iter = path.intermediate_certificates();
+    while let Some(inter) = iter.next_back() {
+        reversed.push(inter);
+    }
+
+    assert_eq!(inters.len(), reversed.len());
```

Comment:
not completely clear what this is testing, or why the reversing of the path intermediates is specific to the netflix test

Reply:
Removed this test, will add some other stuff.